src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx

```diff
@@ -152,6 +152,7 @@ export default function EnvTriggerView({ filteredAppIds, isVirtualEnv }: AppGrou
             setPageViewType(ViewType.LOADING)
             setSelectedAppList([])
             getWorkflowsData()
+            getConfigs()
```

Comment:
Why is this required? It was getting called on basis of url change

Reply:
It was not calling due to handledLocation check. And so called in mounting